pkg/featureflag/featureflag.go

```diff
@@ -84,6 +84,8 @@ var (
 	VSphereCloudProvider = New("VSphereCloudProvider", Bool(false))
 	// SkipEtcdVersionCheck will bypass the check that etcd-manager is using a supported etcd version
 	SkipEtcdVersionCheck = New("SkipEtcdVersionCheck", Bool(false))
+	// ConfigurableRollingUpdate enables the RollingUpdate strategy configuration settings
+	ConfigurableRollingUpdate = New("ConfigurableRollingUpdate", Bool(false))
```

Comment:
I think it would be a good idea to have a featureflag until we have the whole set of features in. Then we might consider whether we want to consolidate some of the settings.